src/actions/braze/braze.ts

```diff
@@ -0,0 +1,172 @@
+import * as req from "request-promise-native"
+import * as url from "url"
+
+import * as Hub from "../../hub"
+
+export enum BrazeConfig {
```

Comment:
Doesn't look like this needs to be exported.

Reply:
fixed thanks